ou_dedetai/gui.py

```diff
@@ -69,59 +89,47 @@ def __init__(self, root, **kwargs):
-            self.releasevar.set(self.logos_release_version)
-
-        # release check button
-        self.release_check_button = Button(self, text="Get Release List")
-        self.release_check_button.state(['disabled'])
+        if app.conf._raw.faithlife_product_release:
+            self.release_dropdown['values'] = [app.conf._raw.faithlife_product_release]
+            self.releasevar.set(app.conf._raw.faithlife_product_release)
 
         # Wine row.
         self.wine_label = Label(self, text="Wine exe: ")
         self.winevar = StringVar()
         self.wine_dropdown = Combobox(self, textvariable=self.winevar)
         self.wine_dropdown.state(['readonly'])
         self.wine_dropdown['values'] = []
-        if self.wine_exe:
-            self.wine_dropdown['values'] = [self.wine_exe]
-            self.winevar.set(self.wine_exe)
-        self.wine_check_button = Button(self, text="Get EXE List")
-        self.wine_check_button.state(['disabled'])
+        # Conditional only if wine_binary is actually set, don't prompt if it's not
+        if self.app.conf._raw.wine_binary:
+            self.wine_dropdown['values'] = [self.app.conf.wine_binary]
+            self.winevar.set(self.app.conf.wine_binary)
 
         # Winetricks row.
         self.tricks_label = Label(self, text="Winetricks: ")
         self.tricksvar = StringVar()
         self.tricks_dropdown = Combobox(self, textvariable=self.tricksvar)
         self.tricks_dropdown.state(['readonly'])
         values = ['Download']
-        if self.winetricksbin:
-            values.insert(0, self.winetricksbin)
+        if app.conf._raw.winetricks_binary:
+            values.insert(0, app.conf._raw.winetricks_binary)
         self.tricks_dropdown['values'] = values
         self.tricksvar.set(self.tricks_dropdown['values'][0])
 
         # Fonts row.
         self.fonts_label = Label(self, text="Install Fonts: ")
-        self.fontsvar = BooleanVar(value=1-self.skip_fonts)
+        self.fontsvar = BooleanVar(value=not self.app.conf.skip_install_fonts)
         self.fonts_checkbox = Checkbutton(self, variable=self.fontsvar)
 
         # Skip Dependencies row.
         self.skipdeps_label = Label(self, text="Install Dependencies: ")
-        self.skipdepsvar = BooleanVar(value=1-self.skip_dependencies)
+        self.skipdepsvar = BooleanVar(value=not self.app.conf.skip_install_system_dependencies) #noqa: E501
         self.skipdeps_checkbox = Checkbutton(self, variable=self.skipdepsvar)
 
         # Cancel/Okay buttons row.
         self.cancel_button = Button(self, text="Cancel")
         self.okay_button = Button(self, text="Install")
         self.okay_button.state(['disabled'])
 
-        # Status area.
-        s1 = Separator(self, orient='horizontal')
-        self.statusvar = StringVar()
-        self.status_label = Label(self, textvariable=self.statusvar)
-        self.progressvar = IntVar()
-        self.progress = Progressbar(self, variable=self.progressvar)
-
```

Comment:
We have a couple options:
- Close the installer window after it started the install (and gray out the install button), showing the progress on the control window. This may be useful if we ever have an "express install button" where the user doesn't select any preferences, it just goes with the default
- Make it look how it did before, using a conditional to figure out whether or not to send status messages to the installer window or control window

Reply:
Probably should be inversed: install or advanced install.

Reply:
Implemented advanced install button